src/ray/gcs/gcs_server/gcs_placement_group_scheduler.h

```diff
@@ -227,32 +293,22 @@ class GcsPlacementGroupScheduler : public GcsPlacementGroupSchedulerInterface {
   /// Factory for producing new clients to request leases from remote nodes.
   ReserveResourceClientFactoryFn lease_client_factory_;
 
+  /// A vector to store all the schedule strategy.
+  std::vector<std::shared_ptr<GcsScheduleStrategy>> scheduler_strategies_;
+
   /// Map from node ID to the set of bundles for whom we are trying to acquire a lease
   /// from that node. This is needed so that we can retry lease requests from the node
   /// until we receive a reply or the node is removed.
   /// TODO(sang): We don't currently handle retry.
   absl::flat_hash_map<ClientID, absl::flat_hash_set<BundleID>>
       node_to_bundles_when_leasing_;
 
-  /// Map from node ID to the set of bundles. This is needed so that we can reschedule
-  /// bundles when a node is dead.
-  absl::flat_hash_map<ClientID,
-                      absl::flat_hash_map<BundleID, std::shared_ptr<BundleSpecification>>>
-      node_to_leased_bundles_;
-
-  /// A vector to store all the schedule strategy.
-  std::vector<std::shared_ptr<GcsScheduleStrategy>> scheduler_strategies_;
-
   /// Set of placement group that have lease requests in flight to nodes.
   /// It is required to know if placement group has been removed or not.
   absl::flat_hash_set<PlacementGroupID> placement_group_leasing_in_progress_;
 
-  /// A map from placement group id to bundle locations.
-  /// It is used to destroy bundles for the placement group. When we reschedule bundles,
-  /// we can get the location of other bundles from here.
-  /// NOTE: It is a reverse index of `node_to_leased_bundles`.
-  absl::flat_hash_map<PlacementGroupID, std::shared_ptr<BundleLocations>>
-      placement_group_to_bundle_locations_;
+  /// Index to lookup bundle locations of node or placement group
```

Comment:
```suggestion
  /// Index to lookup bundle locations of node or placement group.
```

Reply:
Thanks for all these catch!!! Will be more careful about commenting next time.